src/pyaro/timeseries/AutoFilterReaderEngine.py

```diff
@@ -31,7 +31,7 @@ def supported_filters(cls) -> list[Filter]:
 
         :return: list of filters
         """
-        supported = "variables,stations,countries,bounding_boxes,duplicates,time_bounds,time_resolution,flags,time_variable_station,altitude,relaltitude".split(
+        supported = "variables,stations,countries,bounding_boxes,duplicates,time_bounds,time_resolution,flags,time_variable_station,altitude,relaltitude,valleyfloorrelativealtitudefilter".split(
```

Comment:
none of the filters is named filter, and _ is used, so I would suggest `valleyfloor_relaltitude` (since we have relatltitude already anyway).